<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigUi</class>
 <widget class="QWidget" name="ConfigUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Sans Serif</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>1760</x>
     <y>990</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_CfgFile">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
    </font>
   </property>
   <property name="text">
    <string>Config File :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Modules">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
    </font>
   </property>
   <property name="text">
    <string>Modules :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_InImg">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
    </font>
   </property>
   <property name="text">
    <string>Input Image :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_OutDir">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
    </font>
   </property>
   <property name="text">
    <string>Output Directory :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_BackupDir">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
    </font>
   </property>
   <property name="text">
    <string>Backup Directory :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>420</y>
     <width>301</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
    </font>
   </property>
   <property name="title">
    <string>Network Setting</string>
   </property>
   <widget class="QPushButton" name="btnNetworkApply">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
     </font>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioDhcp">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
     </font>
    </property>
    <property name="text">
     <string>DHCP</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioStatic">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
     </font>
    </property>
    <property name="text">
     <string>Static</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Gateway">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
     </font>
    </property>
    <property name="text">
     <string>Gateway :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtDns">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtIpAddress">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtNetmask">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_IP">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
     </font>
    </property>
    <property name="text">
     <string>IP Address :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Netmask">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
     </font>
    </property>
    <property name="text">
     <string>Netmask :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtGateway">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_Dns">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
     </font>
    </property>
    <property name="text">
     <string>DNS :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="edtCfgFile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtImageFile">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>180</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtOutDir">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtBackupDir">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>280</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpenImage">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>180</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
    </font>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpenOutDir">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>230</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
    </font>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBackUpDir">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>280</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
    </font>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>1630</x>
     <y>990</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>420</y>
     <width>651</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Server</string>
   </property>
   <widget class="QLabel" name="label_SvrIP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
     </font>
    </property>
    <property name="text">
     <string>IP Address :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtSvrIpAddress">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_SvrIP_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
     </font>
    </property>
    <property name="text">
     <string>Target Directory :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_SvrIP_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
     </font>
    </property>
    <property name="text">
     <string>Local Mount Point :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtSvrTargetDir">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>451</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtMountDir">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>451</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMountTest">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>170</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Mount Test</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="edtModules">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>btnExit</zorder>
  <zorder>label_CfgFile</zorder>
  <zorder>label_Modules</zorder>
  <zorder>label_InImg</zorder>
  <zorder>label_OutDir</zorder>
  <zorder>label_BackupDir</zorder>
  <zorder>edtCfgFile</zorder>
  <zorder>edtImageFile</zorder>
  <zorder>edtOutDir</zorder>
  <zorder>edtBackupDir</zorder>
  <zorder>btnOpenImage</zorder>
  <zorder>btnOpenOutDir</zorder>
  <zorder>btnBackUpDir</zorder>
  <zorder>btnSave</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>edtModules</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
